src/ListMatch.cc

```diff
@@ -28,17 +28,17 @@ namespace jointcal {
 static double sqr(double x) { return x * x; }
 
 MatchConditions::MatchConditions() { /* default values */
-    NStarsL1 = 70;
-    NStarsL2 = 70;
-    MaxTrialCount = 4;
-    NSigmas = 3.;
-    MaxShiftX = 50;
-    MaxShiftY = 50;
-    SizeRatio = 1;
-    DeltaSizeRatio = 0.1 * SizeRatio;
-    MinMatchRatio = 1. / 3.;
-    PrintLevel = 0;
-    Algorithm = 2;
+    nStarsList1 = 70;
+    nStarsList2 = 70;
+    maxTrialCount = 4;
+    nSigmas = 3.;
+    maxShiftX = 50;
+    maxShiftY = 50;
+    sizeRatio = 1;
+    deltaSizeRatio = 0.1 * sizeRatio;
+    minMatchRatio = 1. / 3.;
+    printLevel = 0;
+    algorithm = 2;
```

Comment:
These should really go in the intializer list, but again another ticket (cleaning up constructors).